parlai/crowdsourcing/tasks/turn_annotations_static/webapp/src/components/task_components.jsx

```diff
@@ -198,9 +216,18 @@ function SubtaskSubmitButton({ subtaskIndex, numSubtasks, onSubtaskSubmit }) {
   )
 }
 
-function ChatMessage({ text, agentIdx, annotationQuestion, annotationBuckets, turnIdx, doAnnotateMessage, askReason, responseField, onUserInputUpdate }) {
+function ChatMessage({ text, agentIdx, annotationQuestion, annotationBuckets, turnIdx, doAnnotateMessage, askReason, responseField, speakerLabel, onUserInputUpdate }) {
   var extraElements = '';
   var responseInputElement = '';
+  if (speakerLabel == null) {	
+    speakerLabel = agentIdx % 2 == 0 ? 'YOU' : 'THEM'	
```

Comment:
never mind I found that. I was a bit concerned about this line. Others looks fine. How is what if  `agentIdx` is not set.

Reply:
Maybe leave a comment if it's obtuse?

Reply:
good call out @jxmsML `turnIdx` was used previously instead of `agentIdx` [here](https://github.com/facebookresearch/ParlAI/blob/master/parlai/crowdsourcing/tasks/turn_annotations_static/webapp/src/components/task_components.jsx#L236) so it makes sense to keep it consistent. Also `turnIdx` is never null so we're good there.